doc/mcuboot/wrapper.rst

```diff
@@ -1,23 +1,29 @@
-.. _mcuboot:
+.. _mcuboot_wrapper:
 
 MCUboot documentation
 #####################
 
+This documentation is based on the official `MCUboot <https://mcuboot.com>`_ documentation, but focuses on the use of MCUboot in the nRF Connect SDK.
```

Comment:
```suggestion
This documentation is based on the official `MCUboot <https://mcuboot.com>`_ documentation, but focuses on the use of MCUboot in nRF Connect SDK.
```
in |NCS|?

Reply:
We don't have shortcuts in this doc set, so |NCS| won't work.